jscripts/bbcodes_sceditor.js

```diff
@@ -182,7 +180,7 @@ $(function ($) {
 				};
 
 			for (var i = 1; i <= 7; i++)
-				content.append($('<a class="sceditor-fontsize-option" data-size="' + i + '" href="#"><font size="' + i + '">' + i + '</font></a>').on('click', clickFunc));
+				content.append($('<a class="sceditor-fontsize-option" data-size="' + i + '" href="#"><font style="font-size: ' + mybbCmd.fsStr[i-1] + '">' + i + '</font></a>').on('click', clickFunc));
```

Comment:
This change isn't relevant to the fix of the behaviour, it just displays the fonts in the drop down in the same size as the text in the editor for that size.